qgreenland/config/helpers/layers/lonlat.py

```diff
@@ -45,13 +44,14 @@ def _make_lonlat_layer(
             asset=asset,
         ),
         steps=[
-            *ogr2ogr(
```

Comment:
Maybe we should go back to using `ogr2ogr` function now that we only need to do one call; then we can pass the `-where` arguments and `-segmentize` arguments to `ogr2ogr_args`.